test/integration/openshift_health_checker/preflight/playbooks/package_update_dep_missing.yml

```diff
@@ -0,0 +1,24 @@
+---
+- include: ../../setup_container.yml
+  vars:
+    image: preflight-aos-package-checks
+    l_host_vars:
+      openshift_deployment_type: openshift-enterprise
+      openshift_release: 3.2
+
+- name: Fails when a dependency required for update is missing
+  hosts: all
+  roles:
+    - openshift_health_checker
+  tasks:
+    - block:
+
+        - include: tasks/enable_repo.yml
+          vars: { repo_name: "break-yum" }
+
+        - action: openshift_health_check
+          args:
+            checks: [ 'package_update' ]
+
+      always:  # destroy the container whether check passed or not
+        - include: ../../teardown_container.yml
```

Comment:
Agreed about teardown, I'm pretty sure that can go in a callback plugin.

For setup, you're always going to need some way to specify the various parameters for the host, which is most of what's copy-pasted. It could be a little cleaner if there were a setup action or something like that.